graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH5.kt

```diff
@@ -0,0 +1,126 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctionalElements
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+@BEH(GraphBehaviours.GetShortestPath, GraphFunctionalElements.GraphFunction, """
+
+""")
+class BEH5 : GraphScenario() {
+
+    @Eac("""
+        Given a graph that contains at least path from a beginning vertex to an ending vertex,
+        the graph will have a shortest path between these vertices
```

Comment:
❓I don't understand what you're trying to say with this... do you mean something like:
> if there is at least one path between the start and end vertex in the given graph, then a shortest path exists

Note that this phrasing assumes that "start vertex" and "end vertex" have been defined as arguments of the function in the behaviour description.